parm/config/config.fv3

```diff
@@ -47,8 +47,6 @@ case $case_in in
         export layout_y=2
         export layout_x_gfs=3
         export layout_y_gfs=2
-        export npe_wav=16
```

Comment:
Where was this moved to instead?

Reply:
All PE stuff was moved to config.resources.

Reply:
Why is PE stuff there and Threading here?
Either both go there or both stay here.

Reply:
Only the wave PE count was moved. The atm PE count is derived in config.resources from the layout, threads, and writegroup settings that are currently still in config.fv3. There is no corresponding case block in config.resources, so one would have to be added to move nth_fv3 (or more) there. I'm also think it is a bit fuzzy as to what we should classify as a "resource" here vs a model parameter. The nth_fv3* goes, but what about the grid decomposition or the write groups?

This is another change we can tackle, but would recommend postponing it to a future PR.

Reply:
Ok. In that case please keep the current behavior. i.e. pes and threads in the same file.

Reply:
The number of wave PEs aren't (well, shouldn't be) related to the atm resolution and shouldn't have been in this file in the first place. That's the only thing that was moved out (and now it depends on the $waveGRD and not $CASE). So it wasn't just moving it, it was changing how wave PEs are calculated to something sensible.